### PR TITLE
Fixed bad missing performance counters error message

### LINKED ISSUE #4230: Bad missing performance counters error message
> 2016-10-12 14:53:51.9052|FATAL| 1|Host.Program|MainNServiceBus performance counter for 'Critical Time' is not set up correctly. To rectify this problem download the latest powershell commandlets from http://www.particular.net downloads page.

There are no PowerShell commandlets at http://particular.net/downloads

I'm not sure exactly how the error message should change. The info I found to get things working is at https://docs.particular.net/nservicebus/operations/performance-counters#installing-counters but I'm not sure we want to deep link.

### PR DESCRIPTION
**Pull request by adamralph**
Fixes #4230

### CODE REVIEW

File: src/NServiceBus.Core/Performance/Counters/PerformanceCounterHelper.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ static bool TryToInstantiatePerformanceCounter(string counterName, string instan
                 throw new Exception($"The endpoint name ('{instanceName}') is too long (longer then {(int) sbyte.MaxValue}) to register as a performance counter instance name. Reduce the endpoint name.");
             }
 
-            var message = $"NServiceBus performance counter for '{counterName}' is not set up correctly. To rectify this problem download the latest powershell commandlets from http://www.particular.net downloads page.";
+            var message = $"NServiceBus performance counter for '{counterName}' is not set up correctly. To rectify this problem download consult the documentation regarding performance counters at https://docs.particular.net.";
```
Comment (timbussmann) [inline src/NServiceBus.Core/Performance/Counters/PerformanceCounterHelper.cs]: > download consult the documentation

sounds wrong ;)

This seems like the first time we really point to the docs url. I only know statements like "consult the documentation". Should we be consistent with that?

**Comment by adamralph** [comment]
@timbussmann I've fixed the typo, by amending the commit.

**Comment by timbussmann** [comment]
@adamralph 👍 

to me it sounds like there should also be a comma:
> To rectify this problem, consult the documentation regarding performance counters

isn't that the case?

**Comment by adamralph** [comment]
@timbussmann not 100% sure, but it reads well with the comma, so I added it with another amend.

**Comment by timbussmann** [comment]
I'm good with the current state then. Still no big fan of the url in the message, but as this was already in there, this isn't really a blocker from my perspective (still something we should probably address).

**Comment by andreasohlund** [comment]
@Particular/docs-maintainers are you ok with the link to docs in the exception message?

**Comment by adamralph** [comment]
👍 to a phrase vs URL. I'll amend the commit.

**Comment by adamralph** [comment]
Amended.

**Pull request merged**